payjoin-cli/src/app.rs

```diff
@@ -539,7 +576,7 @@ impl App {
             )?;
 
         // Receive Check 1: Can Broadcast
-        let proposal = proposal.check_can_broadcast(|tx| {
+        let proposal = proposal.check_broadcast_suitability(None, |tx| {
```

Comment:
this will solve https://github.com/payjoin/rust-payjoin/pull/113#issuecomment-1854686668 ?